### ISSUE TITLE
na2ni: add ZeroMQ output

### ISSUE THREAD

**Issue by lars18th** [piratfm/eti-tools#16]
Hi,

The current implementation of the tool `na2ni` lacks the ZeroMQ output.
So, in order to have the option to interchange the tools (`na2ni` with `edi2eti`) will be desirable to support also this other output.

Example: `usage: ./na2ni [--no-fec] [-i <inputfile>] [-o <outputfile|zeromq-uri>]`

It has sense? You agree?

**Comment by piratfm** [comment]
Yep, that makes sense. ZeroMQ will be in server mode, so few modulators can connect to it and play that stream.

**Comment by lars18th** [comment]
Hi @piratfm ,

> Yep, that makes sense.

Great to hear it! :smile: 

> ZeroMQ will be in server mode

However, why _only_ "server" mode? I feel it be more beneficial to have "client" and "server" options:
- In server mode different clients can connect.
- In client mode it can push (like now does `edi2eti`) to any modulator/player.

So, I suggest to first replicate the same functionality already implemented in `edi2eti`, and then expand it to the "server mode" (in both tools). You agree?

**Comment by piratfm** [comment]
> In client mode it can push

It can't.

**Comment by lars18th** [comment]
Hi @piratfm ,

>> In client mode it can push

> It can't.

Jus to know it... why not? It's not doing in this way the `edi2eti` tool? What's the difference?

**Comment by piratfm** [comment]
ni2http uses connect method because it's connecting to ODR-DabMux and can stream single station to Your own multiplexor, so it's some kind of re-multiplexing feature.

**Comment by lars18th** [comment]
Hi @piratfm ,

> But adding unneeded features is against KISS policy.

I don't like to propose unneeded features. Your "eti-tools" are useful for a lot of projects. My proposal is only related to support both "connect" and "bind" modes because this has a lot flexibility about the other end.

However, I don't want to produce any interference in your work. Please, implement what you prefer.
I hope only that you will appreciate my suggestions.
